src/media/css/buttons.styl

```diff
@@ -198,6 +211,17 @@ a.button {
         margin-top: 0;
         width: fit-content;
     }
+    +website-detail() {
```

Comment:
probably better to split out .mkt-website-button into its own group of rules so it doesn't apply to mkt-app-button